Source/Bogus/Binder.cs

```diff
@@ -94,5 +95,18 @@ public Binder(BindingFlags bindingFlags)
          //ClassA.Value.
          return group.ToDictionary(k => k.Key, g => g.First());
       }
+
+      private static MemberInfo GetPropertyOnBaseType(MemberInfo member, Type reflectedType)
```

Comment:
Note: I passed the reflected type here, because the `ReflectedType` property isn't available in netstandard1.3.